app/src/main/java/org/feature/fox/coffee_counter/data/repository/UserRepository.kt

```diff
@@ -46,6 +55,10 @@ class UserRepository @Inject constructor(
         userDao.deletePurchase(purchase)
     }
 
+    override suspend fun deleteImage(image: Image) {
+        userDao.deleteImage(image)
+    }
+
```

Comment:
Pls add Db at the end of the method name to stay consistent and provide better Overview